### DISCUSSION TITLE
Grid 3x3 or 4x4

### DISCUSSION THREAD

**Discussion by w4t3rf** [FossifyOrg/General-Discussion#254] [Ideas]
Hello,

I quite often use a 3x3 grid to better frame my picture. I found this tool quite useful and it also exists on other apps, phones, or cameras.

Would it be possible to implement grid within the app UI ?? If you mind it useful or important ?

Thank you

**Comment by Aga-C** [comment]
It has already been requested for Gallery: https://github.com/FossifyOrg/Gallery/issues/46.

BTW. Next time, if you want to ask about a specific app, please use discussions in that app's repository.
